### PR TITLE
chore(ci): Update action.yml

### PR DESCRIPTION
**Pull request by breningham**
extractions/setup-just@v2 bumps the version of nodejs used to 20 - should silence the 'node 16 is deprecated' messages

### CODE REVIEW

**Review by ArtikusHG — APPROVED** [review]
Looks good to me, but I'll wait for someone from the core team to approve!

**Review by bsherman — APPROVED** [review]
Sorry, I'd never seen this PR before now.

Thank you for the submission.

**Pull request merged**
